tools/toolchain/scripts/generate_arch_files.sh

```diff
@@ -132,7 +132,7 @@ else
   FC_arch+=" IF_MPI(-fc=${I_MPI_FC}|)"
   LD_arch+=" IF_MPI(-fc=${I_MPI_FC}|)"
   CFLAGS="${G_CFLAGS} -std=c11 -Wall \$(DFLAGS)"
-  CXXFLAGS="${G_CFLAGS} -std=c11 -Wall \$(DFLAGS)"
+  CXXFLAGS="${G_CFLAGS} -std=c++11 -Wall \$(DFLAGS)"
```

Comment:
In CMake you set `set(CMAKE_CXX_STANDARD 14)`.
```suggestion
  CXXFLAGS="${G_CFLAGS} -std=c++14 -Wall \$(DFLAGS)"
```
?